processor/workers.go

```diff
@@ -692,6 +691,33 @@ func fileProcessorWorker(input chan *FileJob, output chan *FileJob) {
 				atomic.CompareAndSwapInt64(&startTime, 0, makeTimestampMilli())
 
 				fileStartTime := makeTimestampNano()
+
+				// Needs to always run to ensure the language is set
+				determineLanguage(res)
+
+				// If the type is #! we should check to see if we can identify
+				if res.Language == SheBang {
+					cutoff := 200
+					if len(res.Content) > cutoff {
```

Comment:
Is this back to front? Shouldn't it be checking for content length *less than* the cutoff length?

Reply:
Yep 100% correct. Fixed.